python/cuml/dask/common/part_utils.py

```diff
@@ -23,7 +24,7 @@
 
 from dask.array.core import Array as daskArray
 from dask_cudf.core import DataFrame as dcDataFrame
-from dask_cudf.core import Series as dcSeries
+from dask_cudf.core import Series as daskSeries
```

Comment:
I like the promotion from dcSeries to daskSeries... can you do the same for dcDataFrame? Then all 3 imports are consistent here.